src/Cake.Core/Scripting/ScriptConventions.cs

```diff
@@ -81,6 +81,9 @@ public IReadOnlyList<Assembly> GetDefaultAssemblies(DirectoryPath root)
             result.Add(typeof(System.Xml.XmlReader).GetTypeInfo().Assembly); // System.Xml
             result.Add(typeof(System.Xml.Linq.XDocument).GetTypeInfo().Assembly); // System.Xml.Linq
             result.Add(typeof(System.Data.DataTable).GetTypeInfo().Assembly); // System.Data
+
+            // This is just to please Roslyn when running under Mono. See issue https://github.com/dotnet/roslyn/issues/19364
+            result.Add(_loader.Load(new AssemblyName("System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"))); // System.Runtime
```

Comment:
@patriksvensson I'd say it's always safe to have `System.Runtime`. It's really just a facade referencing `mscorlib`, `System`, `System.ComponentModel.Composition` and `System.Core`.